src/hooks/__tests__/useGasPrice.test.ts

```diff
@@ -170,6 +170,56 @@ describe('useGasPrice', () => {
     expect(result.current[0]?.maxPriorityFeePerGas?.toString()).toEqual('4975')
   })
 
+  it('should be able to set a fixed EIP 1559 gas price', async () => {
+    ;(useCurrentChain as jest.Mock).mockReturnValue({
+      chainId: '10',
+      gasPrice: [
+        {
+          type: 'fixed1559',
+          maxFeePerGas: '100000000',
+          maxPriorityFeePerGas: '100000',
+        },
+      ],
+      features: ['EIP1559'],
+    })
+
+    const { result } = renderHook(() => useGasPrice())
+
+    await act(async () => {
+      await Promise.resolve()
+    })
+    // assert the hook is not loading
+    expect(result.current[2]).toBe(false)
+
+    // assert fixed gas price as minimum of 0.1 gwei
+    expect(result.current[0]?.maxFeePerGas?.toString()).toBe('100000000')
+
+    // assert fixed priority fee
+    expect(result.current[0]?.maxPriorityFeePerGas?.toString()).toBe('100000')
```

Comment:
nit: You can also wrap all the `expects` with an `await waitFor()` call instead of having an empty `Promise.resolve()` to achieve the same